kratos/mpi/tests/test_KratosMPICore.py

```diff
@@ -18,6 +18,10 @@
 import test_nodal_elemental_neighbours
 import test_parallel_environment
 
+with KratosUnittest.WorkFolderScope("../../tests", __file__, True):
```

Comment:
This may not work in the copied/installed files, be careful

Reply:
it works, we checked

Reply:
Now it works... I had a hanging node bug from partitioner previously. Now all open mp tests are passing in mpi as well :)